documentation/feature_configuration.md

```diff
@@ -23,9 +23,8 @@ report_sections:
 ### 2. Available Features
 
 Please refer to the default configuration files to explore the available features and their names.
-
-- [Object Detection features](src/data_gradients/config/detection.yaml)
-- [Semantic Segmentation features](src/data_gradients/config/segmentation.yaml)
+- [Object Detection features](https://github.com/Deci-AI/data-gradients/blob/9e3a2eb248f4dc95137d94a7d5596fb27c1de3c1/src/data_gradients/config/detection.yaml) - located in `src/data_gradients/config/detection.yaml`
```

Comment:
While git blob is a great way to access the real of a file, I wouldn't use the doc link because if we ever change the yaml file here - this address will still point to the old file without the changes.

Reply:
We should either use relative path here (with ..) or point directly to the github copy without accessing the blob.
@netanel-haber-deci WDTY? if we use .. will that work with our protections in the doc site?

Reply:
@RanZilberstein you are totally right, the blob is too strict to the location. 

I will change the paths to relative ones - but I need @netanel-haber-deci to verify my changes are working well.